autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -112,10 +112,10 @@ def fit(self, X: Dict[str, Any], y: Any = None) -> torch.utils.data.DataLoader:
         self.train_data_loader = torch.utils.data.DataLoader(
             train_dataset,
             batch_size=min(self.batch_size, len(train_dataset)),
-            shuffle=True,
+            shuffle=False,
             num_workers=X.get('num_workers', 0),
             pin_memory=X.get('pin_memory', True),
-            drop_last=X.get('drop_last', True),
+            drop_last=X.get('drop_last', False),
```

Comment:
Could you explain the intentions behind those changes?

Reply:
`shuffle` from `True` to `False` was an ablation since on the previous version of the cocktails it was turned to False by default. The optimal behavior should be true, you do want the data to be shuffled on each epoch. The `drop_last` was also `True` by default, it should be `False`, we do not want to be dropping data especially when dealing with small datasets and a large batch size.

So all in all, the change to `shuffle` will be reverted and the change to` drop_last` should stay.

Reply:
done